iModelCore/iModelPlatform/DgnCore/DgnDbSchema.cpp

```diff
@@ -546,8 +546,17 @@ ProfileState DgnDb::_CheckProfileVersion() const
 /*---------------------------------------------------------------------------------**//**
 * @bsimethod
 +---------------+---------------+---------------+---------------+---------------+------*/
-BeSQLite::DbResult DgnDb::_OnBeforeProfileUpgrade()
+BeSQLite::DbResult DgnDb::_OnBeforeProfileUpgrade(Db::OpenParams const& params)
     {
+    if (GetBriefcaseId().IsBriefcase())
+        {
+        if (!((const DgnDb::OpenParams&)params).GetAllowDataTransformDuringSchemaUpdate())

```

Comment:
I now see that this subclass already existed - yuk.

Reply:
Fixed